mars/deploy/oscar/tests/test_fault_injection.py

```diff
@@ -48,17 +51,25 @@ async def fault_cluster(request):
         yield client
 
 
-async def create_fault_injection_manager(session_id, address, fault_count, fault_type):
+async def create_fault_injection_manager(session_id, address, fault_count, fault_type, fault_op_types=None):
     class FaultInjectionManager(AbstractFaultInjectionManager):
         def __init__(self):
             self._fault_count = fault_count
 
         def set_fault_count(self, count):
             self._fault_count = count
 
+        def get_fault_count(self):
+            return self._fault_count
+
         def get_fault(self, pos: FaultPosition, ctx=None) -> FaultType:
+            # Check op types if fault_op_types provided.
+            if fault_op_types and type(ctx.get('operand')) not in fault_op_types:
+                return FaultType.NoFault
             if self._fault_count.get(pos, 0) > 0:
                 self._fault_count[pos] -= 1
+                if fault_op_types:
+                    print(f"get_fault {fault_type} {pos} {ctx}")
```

Comment:
Is this line for debug?

Reply:
It's for the fault injection test case to know which fault types are raising. But, I am OK to remove it.

Reply:
The print has been removed.